deploy/newOracle/newOracle.ts

```diff
@@ -0,0 +1,38 @@
+import { DeployFunction } from 'hardhat-deploy/types';
+import yargs from 'yargs';
+
+import { OracleSTEURETHChainlinkArbitrum, OracleSTEURETHChainlinkArbitrum__factory } from '../../typechain';
+
+const argv = yargs.env('').boolean('ci').parseSync();
+
+const func: DeployFunction = async ({ deployments, ethers, network }) => {
+  const { deploy } = deployments;
+  const { deployer } = await ethers.getNamedSigners();
+
+  // const treasury = (await deployments.get(`Treasury`)).address;
+  // Treasury_EUR Arbitrum
+  const treasury = '0x37963F10245e7c3a10c0E9d43a6E617B4Bc8440A';
```

Comment:
Just checked this address and it is notthe treasury, it is the Chainlink feed - which has a odd implementation btw